hack/robot-e2e/ansible.cfg

```
@@ -0,0 +1,6 @@
+[defaults]
+inventory = ${PWD}/inventory.yml
+host_key_checking = False
+
```

```suggestion
[defaults]
inventory = ${PWD}/inventory.yml
host_key_checking = False
stdout_callback = community.general.yaml
```

This should improve the ansible logs, by printing yaml instead of json. You might also need to add the `community.general` collection to the requirements.yml file.